### PR TITLE
[CX2-2112] Working concept

### PR DESCRIPTION
**Pull request by iamlordaubrey**
Implement endpoints to get phone-number and version:
GET https://api.onebox.thezebra.com/mediaalpha-ivr/version/
POST https://api.onebox.thezebra.com/mediaalpha-ivr/phone-number/ along with the payload

**Local Testing**

- GET https://api.onebox.thezebra.com/mediaalpha-ivr/version/
- POST https://api.onebox.thezebra.com/mediaalpha-ivr/phone-number/ along 
with the sample payload in the body: 
{
    "data": {
        "zip": 73301
    },
    "ip": "38.140.212.234",
    "local_hour": 15,
    "phone_num_lease_time": 600,
    "placement_id": null,
    "sub_1": 1,
    "ua": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.106 Safari/537.36 OPR/38.0.2220.41",
    "url": "thezebra.com",
    "version": 17
}

**TODO**

**Developer Checklist**

- [x] Add JIRA ticket# to PR title
- [x] Add reviewers
- [x] Test code locally

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import os
 
-from dotenv import load_dotenv, find_dotenv
-load_dotenv(find_dotenv())
-
-MEDIA_ALPHA_URL = os.getenv("MEDIA_ALPHA_URL")
```
Comment (benweatherman) [inline settings.py]: We still wanna keep this as an environment var

File: serverless.yml [hunk 2]
```diff
@@ -3,21 +3,59 @@ service: media-alpha
...
+  - serverless-offline
   - serverless-python-requirements
+  - serverless-domain-manager
+  - serverless-plugin-git-variables
+
+package:
+  individually: true
+  exclude:
+    - .virtualenv/**
+    - .venv/**
```
Comment (benweatherman) [inline serverless.yml]: I guess we should stick with `.virtualenv` to stay consistent. Let's remove `.env` and `venv`

File: handler.py [hunk 3]
```diff
@@ -3,8 +3,9 @@
 
 import requests
 
-from settings import MEDIA_ALPHA_URL, PAYLOAD_SECRETS
+from settings import PAYLOAD_SECRETS
 
+MEDIA_ALPHA_URL = "https://insurance-test.mediaalpha.com/ivr.json"
```
Comment (benweatherman) [inline handler.py]: I think we don't want to hardcode this

File: .gitignore [hunk 4]
```diff
@@ -16,8 +16,10 @@ var/
 .installed.cfg
 *.egg
 
-.env
 .idea/
+.venv/
```
Comment (benweatherman) [inline .gitignore]: Let's remove `.venv` since we'll standardize on `.virtualenv` everywhere

**Review by benweatherman — COMMENTED** [review]
A few tweaks would be good. Thoughts?

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,5 @@
 certifi==2019.3.9
 chardet==3.0.4
 idna==2.8
-python-dotenv==0.10.1
 requests==2.21.0
-urllib3==1.24.1
+urllib3==1.25.1
```
Comment (benweatherman) [inline requirements.txt]: Do we explicitly need this dependency? I think it'll get pulled in when `requests` gets installed

File: media-alpha-dev-us-east-1-policy.json [hunk 6]
```diff
@@ -0,0 +1,178 @@
+{
```
Comment (benweatherman) [inline media-alpha-dev-us-east-1-policy.json]: Should we check this file in?

**Review by benweatherman — COMMENTED** [review]
I had a few questions and possible tweaks, but this is looking great!

File: requirements.txt (hunk 5 above)
Comment (iamlordaubrey) [reply requirements.txt]: `requests` actually pulls in `certifi`, `chardet`, `idna` and `urllib3`. I could take all those out and leave only `requests` in there.

File: media-alpha-dev-us-east-1-policy.json (hunk 6 above)
Comment (iamlordaubrey) [reply media-alpha-dev-us-east-1-policy.json]: I don't think this is necessary. I'll take it out

File: requirements.txt (hunk 5 above)
Comment (benweatherman) [reply requirements.txt]: Yes, let's do that. We should only specify the top-level requirement, then let it determine which packages to install

**Review by benweatherman — APPROVED** [review]
🐐

**Comment by benweatherman** [comment]
@iamlordaubrey do you have access to the settings tab? you should add iz2-tech as admins

**Pull request merged**